msticpy/data/queries/graph_alerts.yaml

```diff
@@ -12,6 +12,13 @@ defaults:
         description: Path name
         type: str
         default: '/security/alerts'
+      query_project:
```

Comment:
This looks wrong - shouldn't need a query_project and it should be an OData expression, not a Kql one.

Reply:
No idea where this came from, must have accidentally copied it from kql_sent_alert and committed by mistake.